jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/test/java/org/jbpm/workbench/cm/client/list/CaseInstanceListPresenterTest.java

```diff
@@ -73,6 +82,10 @@ public void init() {
         when(caseManagementService.getCaseInstances(any(CaseInstanceSearchRequest.class))).thenReturn(caseInstanceSummaryList);
         presenter.setCaseService(caseService);
         when(view.getCaseInstanceSearchRequest()).thenReturn(new CaseInstanceSearchRequest());
+        doAnswer((i) ->{
+            presenter.refreshData();
```

Comment:
I gave a second thought on this test class and I think in here you're actually mocking the bit that needs to be tested.
It would be better if you create a new test class for `CaseInstanceListSearchViewImpl` and then mock the presenter to verify it was actually called.

Reply:
ok, So Do I  need create test for all method (publlic ) of  CaseInstanceListSearchViewImpl?